src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Directory.Move.cs

```diff
@@ -92,6 +126,74 @@ private void DirectoryMove_DifferentWatchedDirectory(WatcherChangeTypes eventTyp
             }
         }
 
+        private void DirectoryMove_Multiple_FromWatchedToUnwatched(int filesCount, bool skipOldEvents)
+        {
+            Assert.True(filesCount > 0);
+
+            using (var watchedTestDirectory = new TempDirectory(GetTestFilePath()))
+            using (var unwatchedTestDirectory = new TempDirectory(GetTestFilePath()))
```

Comment:
nit: consider a using declaration to avoid double-indenting the entire method body.